prow/jobs/kyma/kyma-release-candidate.yaml

```diff
@@ -0,0 +1,61 @@
+job_template: &job_template
+  skip_report: false
+  decorate: true
+  path_alias: github.com/Szymongib/kyma # TODO - for script testing
+  max_concurrency: 10
```

Comment:
Do we need 10 of these running in parallel?

Reply:
Not really, I reduced it to 1